lib/src/test/java/io/ably/lib/test/realtime/RealtimeResumeTest.java

```diff
@@ -120,7 +120,7 @@ public void resume_simple() {
 
 			/* wait for the publish callback to be called */
 			ErrorInfo[] errors = msgComplete1.waitFor();
-			assertTrue("Verify success from all message callbacks", errors.length == 0);				
+			assertTrue("Verify success from all message callbacks", errors.length == 0);
```

Comment:
I assume that your IDE made this change automatically. We do want these changes, but for future reference, please make any such changes (whitespace-only, or style-only) in a separate commit, so we can more easily review the commits that do have code changes.